components/camel-cxf/camel-cxf-rest/src/main/docs/cxfrs-component.adoc

```diff
@@ -25,7 +23,7 @@ for this component:
 -------------------------------------------------------------------------------------
 <dependency>
    <groupId>org.apache.camel</groupId>
-   <artifactId>camel-cxf</artifactId>
+   <artifactId>camel-cxf-rest</artifactId>
```

Comment:
I think this was wrong, but I am not that well versed in CXF. Please, would you mind a review here @davsclaus / @oscerd ?

Reply:
You're right